pytest_timeouts.py

```diff
@@ -172,7 +158,10 @@ def fetch_marker_timeout(item, name):
         def get_fixture_scope(item):
             return item._fixtureinfo.name2fixturedefs[
                 item._fixtureinfo.names_closure[0]][0].scope
-        markers = TimeoutsPlugin.get_markers(item, name)
+
+        get_marker_func = getattr(item, "iter_markers",
+                                  getattr(item, "get_marker", None))
```

Comment:
None as default is bad idea.
Result of invoke `None`
`'NoneType' object is not callable`, debug it won't be easy.
Helper function with raise would be more user friendly.

Reply:
Fixed